lib/mutant/color.rb

```diff
@@ -24,7 +24,7 @@ def format(text)
         text
       end
 
-    private
+    private # rubocop:disable AccessModifierIndentation
```

Comment:
This line is disabled because the class definition begins at column 12 so rubocop tries to indent this `private` to column 12 as well